install.sh

```diff
@@ -2842,34 +1972,75 @@ FIREEDGE_DEV_FILES="src/fireedge/src \
 
 FIREEDGE_ETC_FILES="src/fireedge/etc/fireedge-server.conf"
 
-#----------------------------------------------------------------------------
-# FireEdge Provision files
-#----------------------------------------------------------------------------
-
-FIREEDGE_PROVISION_ETC="src/fireedge/etc/provision/provision-server.conf"
-
-FIREEDGE_PROVISION_ETC_PROVIDERS="src/fireedge/etc/provision/providers.d/aws.yaml \
-                                src/fireedge/etc/provision/providers.d/digitalocean.yaml \
-                                src/fireedge/etc/provision/providers.d/dummy.yaml \
-                                src/fireedge/etc/provision/providers.d/google.yaml \
-                                src/fireedge/etc/provision/providers.d/equinix.yaml \
-                                src/fireedge/etc/provision/providers.d/vultr_metal.yaml \
-                                src/fireedge/etc/provision/providers.d/vultr_virtual.yaml"
-
 #----------------------------------------------------------------------------
 # FireEdge Sunstone files
 #----------------------------------------------------------------------------
 
 FIREEDGE_SUNSTONE_ETC="src/fireedge/etc/sunstone/sunstone-server.conf \
-                       src/fireedge/etc/sunstone/sunstone-views.yaml"
-
-FIREEDGE_SUNSTONE_ETC_VIEW_ADMIN="src/fireedge/etc/sunstone/admin/cluster-tab.yaml \
-                                src/fireedge/etc/sunstone/admin/host-tab.yaml \
-                                src/fireedge/etc/sunstone/admin/vm-tab.yaml \
-                                src/fireedge/etc/sunstone/admin/vm-template-tab.yaml"
-
-FIREEDGE_SUNSTONE_ETC_VIEW_USER="src/fireedge/etc/sunstone/user/vm-tab.yaml \
-                                src/fireedge/etc/sunstone/user/vm-template-tab.yaml"
+                       src/fireedge/etc/sunstone/tab-manifest.yaml \
+                       src/fireedge/etc/sunstone/default-labels.yaml \
+                       src/fireedge/etc/sunstone/remotes-config.yaml"
+
+FIREEDGE_SUNSTONE_ETC_PROFILES="src/fireedge/etc/sunstone/profiles/windows_optimized.yaml \
+                                src/fireedge/etc/sunstone/profiles/base.template"
+
+FIREEDGE_SUNSTONE_ETC_VIEW="src/fireedge/etc/sunstone/views/sunstone-views.yaml"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

FIREEDGE_SUNSTONE_ETC_VIEW appears unused. Verify use (or export if used externally).